packages/iov-core/src/integrationtests/atomicswapbnsbns.spec.ts

```diff
@@ -216,47 +182,42 @@ class Actor {
   }
 }
 
-describe("Full atomic swap between bns and bcp", () => {
+describe("Full atomic swap between BNS and BNS", () => {
```

Comment:
This sounds like it might be two chains. Suggest "between two different tokens on BNS"?

Reply:
Done in the latest commit